cpp/src/arrow/compute/api_scalar.h

```diff
@@ -1518,6 +1518,16 @@ ARROW_EXPORT Result<Datum> AssumeTimezone(const Datum& values,
 ARROW_EXPORT Result<Datum> IsDaylightSavings(const Datum& values,
                                              ExecContext* ctx = NULLPTR);
 
+/// \brief LocalTime converts timestamp to timezone naive local timestamp
+///
+/// \param[in] values input to convert to local time
+/// \param[in] ctx the function execution context, optional
+/// \return the resulting datum
+///
+/// \since 12.0.0
+/// \note API not yet finalized
```

Comment:
I was going to suggest removing this but now I see this is everywhere.  Hmm.... :(  I'm not sure how true this is these days.

Reply:
Well, you could see it as an opportunity to sync the API with substrait :D